src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsBackupShare.ps1

```diff
@@ -89,9 +89,9 @@ function Set-AzsBackupShare {
         [securestring]
         $Password,
 
-        [Parameter(Mandatory = $true, ParameterSetName = 'ResourceId')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'ResourceId')]
```

Comment:
Validation in BRP/IBC will be added so that exception will be thrown, when EncryptionKey is not provided here, and neither exists in backend.